app/assets/stylesheets/provider/_flash.scss

```diff
@@ -1,8 +1,12 @@
 #flashWrapper {
-  position: absolute;
-  right: line-height-times(1);
-  text-align: right;
-  @include animation(fadeInLeft 0.5s ease-in-out);
+  @include animation(fadeInRight 0.5s ease-in-out);
+  background-color: $white;
+  box-shadow: 0 0 20px 0 $whiteBoxShadowColor;
+  display: block;
+  margin-top: 1.5rem;
+  padding: 1rem;
+  position: fixed;
+  right: 2rem;
```

Comment:
It needs a z-index of about 1000 to take care of this:
<img width="405" alt="Screenshot 2020-02-07 12 33 02" src="https://user-images.githubusercontent.com/54224/74026281-2311f680-49a6-11ea-9087-3a1b296f721e.png">